src/explore-education-statistics-admin/.env

```diff
@@ -2,5 +2,6 @@ NODE_ENV=development
 PORT=3001
 BROWSER=none
 
+USE_MOCK_API=true
```

Comment:
Whats this doing? And do we need this setting in every environment?

Reply:
Hey @jamescutts ,

Good shout.  This flag only affects the admin non-prototype views that are currently under development, therefore the admin prototypes for user testing and showing the business are unaffected.  The absence of this flag for a particular environment denotes that the environment will use real API calls for admin non-prototype pages, but if this flag is set to true for a local (or demo) environment, a set of simple mock HTTP responses will be used to support the front end.

During development of the admin functionality I see that the frontend dev will outrun the backend dev by a fair way in the earlier weeks, and so I think it's important that we are able to maintain a set of mock data for supporting the front end that the front end consumes as if it were real API calls, which in turn allows me to demo to @richardpopplehive exactly what data we need at any given point.

I think for the time being we can probably get away without adding it to other environments, but maybe prior to the next show and tell, we should have an environment to demo from that is using the mock responses.

Cheers,
Duncan